src/runtime/objmodel.cpp

```diff
@@ -2484,7 +2488,7 @@ Box* callFunc(BoxedFunctionBase* func, CallRewriteArgs* rewrite_args, ArgPassSpe
         }
     }
 
-    std::vector<Box*> varargs;
+    llvm::SmallVector<Box*, 8> varargs;
```

Comment:
We could also potentially disable GC for the duration of `callFunc` :P  There are barely any allocations in here, though I guess we'd have to be careful about the call to pyElements.